test-website/README.md

```diff
@@ -13,3 +13,28 @@ It contains all kind of situations we currently cope with:
 
 - invalid inline CSS
 - invalid rule in a CSS file
+
+
+## How to use locally
+
+Build the docker image
+
+```
+docker build -t test-website .
+```
+
+Start the test website with appropriate environment variables.
+
+```
+docker run -p 8888:80 --rm --name test-website -e SITE_ADDRESS="test-website.local.oviles.info:80, xn--jdacents-v0aqb.local.oviles.info:80" -e STANDARD_HOSTNAME="http:\/\/test-website.local.oviles.info:8888" -e ACCENTUATED_HOSTNAME="http:\/\/jédéacçents.local.oviles.info:8888" test-website
```

Comment:
Port doesn't respect our convention 😉

Reply:
Which convention? The one which is not yet formally approved? 🤣